### PR TITLE
fix(manager sanity): removed unnecessary suspend command

### PR DESCRIPTION
**Pull request by ShlomiBalalis**
## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [x] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

**Comment by fruch** [comment]
@ShlomiBalalis can you fix the commit headline ? s/rmeoved/removed/

**Comment by ShlomiBalalis** [comment]
> @ShlomiBalalis can you fix the commit headline ? s/rmeoved/removed/

Done

**Review by fruch — APPROVED** [review]
LGTM

File: mgmt_cli_test.py [hunk 1]
```diff
@@ -1006,7 +1006,6 @@ def _suspend_and_resume_task_template(self, task_type):
         assert suspendable_task.wait_for_status(list_status=[TaskStatus.RUNNING], timeout=300, step=5), \
             f"task {suspendable_task.id} failed to reach status {TaskStatus.RUNNING}"
         with mgr_cluster.suspend_manager_then_resume(start_tasks=False):
-            mgr_cluster.suspend()
             assert suspendable_task.wait_for_status(list_status=[TaskStatus.STOPPED], timeout=300, step=10), \
                 f"task {suspendable_task.id} failed to reach status {TaskStatus.STOPPED}"
             mgr_cluster.resume(start_tasks=True)
```
Comment (fruch) [inline mgmt_cli_test.py:1011]: is this resume needed ? since the context manager here would resume, isn't it ?

Comment (fruch) [reply mgmt_cli_test.py:1011]: and the assert below needs to go out of the context, isn't it ?

Comment (ShlomiBalalis) [reply mgmt_cli_test.py]: Correct, done.

**Review by fruch — APPROVED** [review]
LGTM

**Review by fgelcer — APPROVED** [review]
LGTM, although i think the context of this change isn't clear...
a task/issue should be linked, and a description should be added as a commit message (and to the PR description), so one can understand the change, without need to open files, read many rows to understand the meaning of this 2 lines change...

**Comment by ShlomiBalalis** [comment]
> LGTM, although i think the context of this change isn't clear... a task/issue should be linked, and a description should be added as a commit message (and to the PR description), so one can understand the change, without need to open files, read many rows to understand the meaning of this 2 lines change...

It was just a duplication, since I moved the suspend/reusme to the context manager but didn't remove them from the test. That is all.

**Pull request merged**